packages/grafana-ui/src/components/Flex/Flex.internal.story.tsx

```diff
@@ -0,0 +1,214 @@
+        ))}
+      </Flex>
+      <p>Align items stretch</p>
+      <Flex direction="row" wrap="wrap" alignItems="stretch" justifyContent="center" gap={2}>
+        <Item color={theme.colors.error.main} height="10em" />
+        <Item color={theme.colors.error.main} />
+        <Item color={theme.colors.error.main} height="3em" />
+        <Item color={theme.colors.error.main} />
+        <Item color={theme.colors.error.main} />
+      </Flex>
+
+      <p>Align items baseline</p>
+      <Flex direction="row" wrap="nowrap" alignItems="baseline" justifyContent="center" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.error.main} text={i + 1} />
+        ))}
+      </Flex>
+      <p>Align items center</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="center" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.error.main} text={i + 1} />
+        ))}
+      </Flex>
+    </div>
+  );
+};
+
+export const JustifyContentExamples: StoryFn<typeof Flex> = () => {
+  const theme = useTheme2();
+  return (
+    <div style={{ width: '600px' }}>
+      <p>Justify Content space-between</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="space-between" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.warning.main} text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content space-around </p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="space-around" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.warning.main} text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content space-evenly</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="space-evenly" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.warning.main} text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content flex-start</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="flex-start" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.warning.main} text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content flex-end</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="flex-end" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.warning.main} text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content center</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="center" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color={theme.colors.warning.main} text={i + 1} />
+        ))}
+      </Flex>
+    </div>
```

Comment:
I think you could loop through the justifyContent values here

Reply:
Same for direction

Reply:
Thank you for feedback, you are right, it is more clean like that, I updated the code and push now. :)